sessions/react-global-state/demo-end/pages/components/AppLayout.js

```diff
@@ -0,0 +1,54 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+import Header from "./Header";
+import Footer from "./Footer";
+
+const StyledHeader = styled.header`
```

Comment:
This is more my personal feeling: can we make the the content inside of StyledHeader, Container and StyledFooter appear vertically centered with a max-width?

In larger screens there is a lot of white space on the right.